dotCMS/src/main/java/com/dotcms/rest/api/v1/experiments/ExperimentsResource.java

```diff
@@ -301,6 +301,25 @@ public ResponseEntitySingleExperimentView end(@Context final HttpServletRequest
         return new ResponseEntitySingleExperimentView(endedExperiment);
     }
 
+    /**
+     * Cancels the future execution of a Scheduled {@link Experiment}. The Experiment needs to be in
+     * {@link Status#SCHEDULED} status to be able to cancel it.
+     */
+
+    @POST
```

Comment:
should not be a put?

Reply:
PUTs are idempotent which is not the case here since this changes the state of the Experiment to DRAFT